Makefile

```diff
@@ -94,3 +94,97 @@ deploy_frontend: deploy_update_repo
 .PHONY: $(REQUIREMENTS_TXT)
 $(REQUIREMENTS_TXT):
 	pipenv lock -r > $@
+
+# ---
+
+BACKEND_SERVICE := expungeservice
+FRONTEND_SERVICE := node
+DB_SERVICE := postgres
+
+# currently, only used for PGDATABASE var in 'dropdb' target
+include config/postgres/client.env
+
+# one step target for a new dev env
+new: up
```

Comment:
this target should be the one-stop shop for new developer environment setup. it takes a little bit to run, here's what is happening:

* pulls images needed: postgres:10-alpine, recordsponge/expungservice:dev, and node:alpine
* launching postgres, waiting for availability, running DB seeding scripts
* installing node_modules for the frontend in a named volume, then launching dev server on 3000

the last one is what takes the longest. watch what is happening in the "background" with `make frontend_logs`.